apps/assessment_registry/dashboard_schema.py

```diff
@@ -0,0 +1,98 @@
+import graphene
+from .models import AssessmentRegistry, MethodologyAttribute
+from .enums import (
+    AssessmentRegistryCoordinationTypeEnum,
+    AssessmentRegistryDataCollectionTechniqueTypeEnum,
+)
+from django.db.models import Count
+
+
+class AssessmentCountType(graphene.ObjectType):
+    coordinated_joint = graphene.Field(AssessmentRegistryCoordinationTypeEnum)
+    coordinated_joint_display = graphene.String()
+    count = graphene.Int()
```

Comment:
Let's use required=True if the values are always available.

For _display, let's use EnumDescription
```suggestion
    coordinated_joint = graphene.Field(AssessmentRegistryCoordinationTypeEnum, required=True)
    coordinated_joint_display = EnumDescription(required=True)
    count = graphene.Int(required=True)
```

Reply:
graphene.String changed into EnumDescription in dashboard_schema